PalasoUIWindowsForms/ImageToolbox/PalasoImage.cs

```
@@ -348,6 +351,11 @@ protected virtual void Dispose(bool disposing)
 					Image.Dispose();
 					Image = null;
 				}
+
+				if (!string.IsNullOrEmpty(LeakedFilePath))
+				{
+					File.Delete(LeakedFilePath);
```

Perhaps swallow any exceptions on this when not in #Debug?